app/controllers/admin/recoveries_controller.rb

```diff
@@ -47,7 +47,7 @@ def sortable_columns
 
   def recovery_display_status_searched
     return StolenRecord::RECOVERY_DISPLAY_STATUS_ENUM.values if params[:search_recovery_display_status] == "all"
-    # Legacy enum issue so excited for TODO: Rails 5 update
+    # TODO: Refactor to incorporate Rails 5 enum changes
     recovery_display_status_parameter = (params[:search_recovery_display_status] || "waiting_on_decision").to_sym
```

Comment:
Same as above — not sure what issue is being referenced here

Reply:
Answered above, and as stated above, I'd prefer to merge this PR and then deal with those updates, since they don't block shipping this (and fewer changes is better than more changes).

Sorry that the string wasn't perfectly consistent, I was figuring I'd just be grepping through, it's nice that you've made it consistent.